cli/src/main.rs

```diff
@@ -27,6 +27,28 @@ async fn main() -> Result<(), color_eyre::Report> {
 
     if let Ok(config_path) = std::env::var("IROHA2_CONFIG_PATH") {
         args.config_path = std::path::PathBuf::from_str(&config_path)?;
+    } else {
+        if !Path::new("config.json").exists() {
```

Comment:
```suggestion
    } 
    if !args.config_path.exists() {
```

Reply:
this would mean that env variables will be read from if `IROHA2_CONFIG_PATH` doesn't exist or `config.json` doesn't exist